include/rocksdb/statistics.h

```diff
@@ -370,6 +370,9 @@ enum Histograms : uint32_t {
   BYTES_DECOMPRESSED,
   COMPRESSION_TIMES_NANOS,
   DECOMPRESSION_TIMES_NANOS,
+  // Number of merge operands passed to the merge operator in user read
```

Comment:
From my reading the code, it doesn't. Maybe I'm wrong. Let me double check that.